draft-iab-privacy-partitioning.md

```diff
@@ -489,6 +489,22 @@ revealed during the redemption context.
 ~~~
 {: #diagram-privacypass title="Diagram of contexts in Privacy Pass"}
 
+Since the redemption context and issuance context are separate connections
+that involve separate entities, they can also be further decoupled by
+running those parts of the protocols at very different times. Clients can
+fetch tokens through the issuance context early, and cache the tokens
+to later use in redemption contexts. This aids in partioning identifiers
```

Comment:
```suggestion
to later use in redemption contexts. This can aid in partitioning identifiers
```

Reply:
"Can" since separation over time doesn't help in the extreme case where there is just one issuance and redemption.